src/commands.ts

```diff
@@ -781,4 +784,136 @@ export async function getVersion() {
+
+    const headers = new Headers();
+    headers.append("Authorization", `Bearer ${authToken}`);
+    headers.append("Content-Type", "application/json");
+    let giftCards: GiftCard[] = [];
+    const treeIndex = (await this.account.getPoolTreeState()).index
+    try {
+        for (let cardIndex = 0; cardIndex < Number(quantity); cardIndex++) {
+            const alias = `${prefix}_${cardIndex}`;
+            const body = JSON.stringify({ "description": `${alias}` });
+            const signupResponse = await fetch(`${cloudUrl}/signup`, {
+                method: 'POST',
+                headers,
+                body
+            });
+            if (signupResponse.status == 401) {
+                throw new Error("not authorized to create new accounts, check admin token in environment variables")
+            } else if (!signupResponse.ok) {
+                throw new Error(`cloud wallet returned bad response ${signupResponse}` )
+            }
+            const signupResponseJson = await signupResponse.json();
+            const cloudId = signupResponseJson.accountId;
+
+            if(!cloudId) throw new Error("sign up response is invalid")
+    
+            const exportResponse = await fetch(`${cloudUrl}/export?id=${cloudId}`);
+
+            if (!exportResponse.ok) throw new Error(`export failed ${exportResponse}`)
+
+            const exportJson = await exportResponse.json();
+            let sk = `0x${exportJson.sk}`;
+    
+            const generateAddressResponse = await fetch(`${cloudUrl}/generateAddress?id=${cloudId}`);
+
+            if (!generateAddressResponse.ok) throw new Error(`generate address failed ${exportResponse}`);
+            const generateAddressResponseJson = await generateAddressResponse.json();
+            const address = generateAddressResponseJson.address;
+            console.log(`generated new account with address: ${address} `);
+    
+            const url = redemptionUrl(sk, treeIndex);
+            const svg = qrcode(url);
+            giftCards.push(new GiftCard(alias, cloudId, sk, address, svg, url));
+        }    
+    
+        let zipUrl = await zip(giftCards);
+        this.echo(`Cards generated, [[!;;;;${zipUrl}]this archive] contains QR codes and summary report.\nSending funds ...`);    
+        const transferRequests:TransferRequest[] = giftCards.map(
+            giftCard =>  {return {
+                destination: giftCard.address,
+                amountGwei:this.account.humanToShielded(cardBalance) 
+            }
+        } );
+        const result = await this.account.transferShielded(transferRequests);
+
+        this.echo(`Funds transfer is done ${result.map((singleTxResult: { jobId: any; txHash: any; }) => {
+            return `[job #${singleTxResult.jobId}]: [[!;;;;${this.account.getTransactionUrl(singleTxResult.txHash)}]${singleTxResult.txHash}]`
+        }).join(`\n     `)}`);
+        
+    } catch (error) {
+        
+        this.echo("process failed with error: ", error)
```

Comment:
```suggestion
        this.echo(`Process failed with error: [[;red;]${error.message}]`);
```

Otherwise an error isn't visible in output for some reasons:
<img width="1669" alt="image" src="https://user-images.githubusercontent.com/1415489/214251688-bdd01160-56fc-4de5-9753-8938f0daf3f7.png">